Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN pip install --require-hashes --no-cache-dir -r requirements/prod.txt
 ########
 # assets builder and dev server
 #
-FROM node:22.15.0-slim AS assets
+FROM node:24-slim AS assets
```

Comment:
I'm also adding more details about the hard pin version rationale in a tracking ticket:
- https://github.com/mozilla/bedrock/issues/16910